fpgalib/jump_table.py

```diff
@@ -27,6 +27,8 @@ def __init__(self, from_addr, to_addr, operation):
         :param int to_addr: the to address (should be 0 if not used)
         :param Operation operation: the operation
         """
+
+        # TODO: Why is this -1 hard coded?
```

Comment:
AFAICT from the docs, there is no restriction on the relationship between SRAM addresses `from_addr` and `to_addr` on a single jump table op. I think this check is just bogus.